v1.1-beta/matrixx-digital-commerce/matrixx-sba-events/cnf-testsuite.yml

```diff
@@ -2,7 +2,7 @@
 helm_directory: "sba-5gc-events"
 release_name: "cnf"
 helm_install_namespace: "matrixx"
-helm_values: "--values /home/dwilmes/.mtx/konstruxx/working/values-939239483.yaml\
+helm_values: "--values ~/.mtx/konstruxx/working/values-939239483.yaml\
   \ --devel"
```

Comment:
@daniel-wilmes @deerowbear Are end-users expected to use the --devel flag?

Reply:
@taylor removed -devel